src/System.Windows.Extensions/src/System/Drawing/ImageConverter.cs

```diff
@@ -63,5 +75,75 @@ public override PropertyDescriptorCollection GetProperties(ITypeDescriptorContex
         }
 
         public override bool GetPropertiesSupported(ITypeDescriptorContext context) => true;
+
+        private unsafe Stream GetBitmapStream(byte[] rawData)
+        {
+            Debug.Assert(rawData != null, "rawData is null.");
+            try
+            {
+                fixed (byte* pByte = rawData)
+                {
+                    IntPtr addr = (IntPtr)pByte;
+
+                    if (addr == IntPtr.Zero)
+                    {
+                        return null;
+                    }
+
+                    // This will be pHeader.signature, but we try to
+                    // do this first so we avoid building the structure when we shouldn't
+                    if (rawData.Length <= sizeof(OBJECTHEADER) || Marshal.ReadInt16(addr) != 0x1c15)
+                    {
+                        return null;
+                    }
+
+                    // The data is in the form of OBJECTHEADER. It's an encoded format that Access uses to push imagesinto the DB.
+                    OBJECTHEADER pHeader = (OBJECTHEADER)Marshal.PtrToStructure(addr, typeof(OBJECTHEADER));
+
+                    // "PBrush" should be the 6 chars after position 12 as well.
+                    if (rawData.Length <= pHeader.headersize + 18 || Encoding.ASCII.GetString(rawData, pHeader.headersize + 12, 6) != "PBrush")
```

Comment:
Can we make these static byte arrays? You're adding a lot of unnecessary allocations here.

``` C#
private static ReadOnlySpan<byte> PBrush => new byte[] { (byte)'P', (byte)'B', (byte)'r', (byte)'u', (byte)'s', (byte)'h' };
```